pkg/api/v1/types.go

```diff
@@ -2131,6 +2131,11 @@ type PodAffinityTerm struct {
 	// for affinity and for RequiredDuringScheduling pod anti-affinity, empty topologyKey is not allowed.
 	// +optional
 	TopologyKey string `json:"topologyKey,omitempty" protobuf:"bytes,3,opt,name=topologyKey"`
+	// MinMatchingPods specifies the minimal number of pod that matching the selector; the term is considered to
```

Comment:
thanks for your comments :). This PR only implements anti-affintiy; but affinity is easier. I'll update PR accordingly.